centreon/src/Core/Common/Infrastructure/Repository/ValuesByPackets.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
```

Comment:
```suggestion
 * Copyright 2005 - 2024 Centreon (https://www.centreon.com/)
```
Just a side note, you don't have to change everything 😅